concordium-sdk/src/main/java/com/concordium/sdk/Client.java

```diff
@@ -313,6 +311,26 @@ public Semver getVersion() {
         return new Semver(versionString);
     }
 
+    /**
+     * Get the list of anonymity revokers in the given block.
+     *
+     * @param blockHash {@link Hash} of the Block at which the Anonymity Revokers need to be fetched.
+     * @return Parsed {@link ImmutableList} of {@link ArInfo}
+     * @throws Exception When the returned Json is NULL.
+     */
+    public ImmutableList<ArInfo> getAnonymityRevokers(Hash blockHash) throws Exception {
+        val res = server()
+                .getAnonymityRevokers(ConcordiumP2PRpc.BlockHash.newBuilder().setBlockHash(blockHash.asHex()).build());
+
+        ArInfo[] array = ArInfo.fromJsonArray(res);
+
+        if (Objects.isNull(array)) {
+            throw new Exception(String.format("Could not get Anonymity Revoker at block %s", blockHash.asHex()));
```

Comment:
This will occur if the block hash supplied was invalid. I suggest using `BlockNotFoundException`.

Reply:
done